Live/LiveEventWithDVR/Program.cs

```diff
@@ -79,522 +74,375 @@
-    // 5) Understand the differences between a LiveOutput and the Asset that it records to.  They are two different concepts.
-    //    A live output can be considered as the "tape recorder" and the Asset is the tape that is inserted into it for recording.
-    // 6) Understand the advanced options such as low latency, and live transcription/captioning support. 
-    //    Live Transcription - https://learn.microsoft.com/azure/media-services/latest/live-event-live-transcription-how-to
-    //    Low Latency - https://learn.microsoft.com/azure/media-services/latest/live-event-latency-reference
+liveEvent = await StartLiveEventAsync(liveEvent);
 
-    // When broadcasting to a live event, please use one of the verified on-premises live streaming encoders.
-    // While operating this tutorial, it is recommended to start out using OBS Studio before moving to another encoder. 
+PrintIngestUrls(liveEvent);
 
-    // Note: When creating a LiveEvent, you can specify allowed IP addresses in one of the following formats:                 
-    //      IpV4 address with 4 numbers
-    //      CIDR address range  
+var streamingLocator = await DemoStreamingAsync(
+    mediaServicesAccount,
+    asset,
+    streamingEndpointName,
+    streamingLocatorName,
+    manifestName,
+    dvrAssetFilterName);
 
-    var allAllowIpRange = new IPRange()
-    {
-        Name = "AllowAll",
-        Address = IPAddress.Parse("0.0.0.0"),
-        SubnetPrefixLength = 0
-    };
+Console.WriteLine("Cleaning up resources, stopping Live Event billing, and deleting live Event...");
+await CleanupResourcesAsync(
+    liveEvent,
+    liveOutput,
+    streamingLocator,
+    asset);
 
-    // Create the LiveEvent Preview IP access control object. 
-    // This will restrict which clients can view the preview endpoint
-    var liveEventPreview = new LiveEventPreview();
-    liveEventPreview.IPAllowedIPs.Add(allAllowIpRange);
+if (mediaServicesMonitor != null)
+{
+    await mediaServicesMonitor.StopAsync();
+}
+
+static async Task<MediaLiveEventResource> CreateLiveEventAsync(
+    MediaServicesAccountResource mediaServicesAccount,
+    string liveEventName)
+{
+    // Creating the LiveEvent - the primary object for live streaming in Media Services.
+    // See the overview - https://learn.microsoft.com/azure/media-services/latest/live-event-concept
 
-    #region NewLiveEvent
     var liveEventData = new MediaLiveEventData(mediaServicesAccount.Get().Value.Data.Location)
     {
-        Description = "Sample LiveEvent from .NET SDK sample",
+        Description = "Sample LiveEvent from the .NET SDK sample",
         UseStaticHostname = true,
         // 1) Set up the input settings for the Live event...
         Input = new LiveEventInput(streamingProtocol: LiveEventInputProtocol.Rtmp)
         {
             StreamingProtocol = LiveEventInputProtocol.Rtmp,
             AccessToken = "acf7b6ef-8a37-425f-b8fc-51c2d6a5a86a",
```

Comment:
Should we mention in a comment here that this GUID is something that is not used for security and can be changed by the user? It's not clear why this magic GUID string is here.

Reply:
Added a comment.